app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -251,6 +270,17 @@ private void navigateTo(Class<? extends GdgActivity> activityClass, Bundle addit
         mDrawerLayout.closeDrawers();
     }
 
+    protected void setNavDrawerItemSelected(String title) {
+        Menu menu = mNavigationView.getMenu();
```

Comment:
Anyways, it would be really good if you can rebase your branch. You can
delete my id implementation.
And what about having unique item ids in the menu? We can make the ids of
special events like 30,31,32 etc instead of 3.

On Mon, Jul 27, 2015, 4:08 PM Marcus Gabilheri <notifications@github.com>
wrote:

> In
> app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java
> <https://github.com/gdg-x/frisbee/pull/421#discussion_r35532902>:
>
> > @@ -251,6 +270,17 @@ private void navigateTo(Class<? extends GdgActivity> activityClass, Bundle addit
> >          mDrawerLayout.closeDrawers();
> >      }
> >
> > +    protected void setNavDrawerItemSelected(String title) {
> > +        Menu menu = mNavigationView.getMenu();
>
> The problem I was having using the ID is that the SPECIAL_EVENT items have
> the same id therefore clicking on Study Jam would check the WTM, etc.. By
> using their title string makes it be able to find.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/421/files#r35532902>.
>